CMakeLists.txt

```diff
@@ -414,11 +414,10 @@ else()
   install(DIRECTORY ${CMAKE_BINARY_DIR}/etc/dictpch DESTINATION ${CMAKE_INSTALL_SYSCONFDIR})
 endif()
 
+file(GLOB ${pcms} "*.pcm")
```

Comment:
I think this doesn't work as the globbing is done before the build and therefore nothing is found. Most pcms are copied across (presumably from another `install` call) but some are missing (such as `std.pcm`).

Reply:
This PR should make sure all the pcms are installed (still WIP):
https://github.com/root-project/root/pull/4763

The missing pcms are not under the control of the build system:
https://github.com/root-project/root/blob/master/core/dictgen/src/rootcling_impl.cxx#L4237-L4253

@oshadura will complete the PR above to have those pcms installed as well.